assets/gulpfile.js

```diff
@@ -53,7 +53,7 @@ gulp.task('build_js', function() {
       }
     }))
     .pipe(addsrc.prepend(['/gulp/js/dev/lib/*.js', '!/gulp/js/dev/lib/jquery.min.js']))
-    .pipe(addsrc.prepend('/gulp/js/dev/lib/jquery.min.js'))
+    .pipe(addsrc.prepend('/gulp/js/dev/lib/jquery.min.js', { allowEmpty: true }))
```

Comment:
why was the allowEmpty necessary?

Reply:
The assets build step failed otherwise. try it and you'll see.